README.md

```diff
@@ -24,11 +24,11 @@ Have you come across any job posting at an open source company? Feel free to add
 - > Note: Keep all the job posts fresh, with a limit of 3 months range
 - > First Job is an example of how to fill it out
 
-| Position Name             | Company           | Tech Stack      | Expiring Date       | Link to apply               |
-| -------------             | -------------     | --------        | ---------           | -----------                 |
-| Frontend Developer        | Google            | MERN            | 3rd Feburary, 2024  | ![Apply](google.com)        |
-|                           |                   |                 |                     |                             |
-|                           |                   |                 |                     |                             |
+| Position Name       | Company           | Tech Stack       | Expiring Date    | Link to apply             |
+| :-------------:     | :-------------:   | :--------:       | :--------------: | :----------:              |
+| Frontend Developer  | OpenZeppelin      | React,Solidity   | -------          | [Apply](shorturl.at/atVW8)|
```

Comment:
A few things:

1. Can you add an expiring date? You can set it to 3 months from now.
2. There should be a space after "React,"
3. I tried the link but it's not working - can you try the raw link instead?

Reply:
I tried the link too, it was not working for me too.

Reply:
updated the links and date